app/src/main/java/com/android/streetworkapp/ui/park/AddEvent.kt

```diff
@@ -64,11 +67,32 @@ import java.util.concurrent.TimeUnit
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun AddEventScreen(
-    navigationActions: NavigationActions
-) { // TODO: add future ParkViewModel and EventViewModel
+    navigationActions: NavigationActions,
+    parkViewModel: ParkViewModel,
+    eventViewModel: EventViewModel,
+    userViewModel: UserViewModel
+) {
 
-  // used until we have the corresponding viewModel TODO: update with new viewModel
-  val event = Event("unknown", "unknown", "unknown", 0, 2, Timestamp(0, 0), "unknown")
+  val eid = eventViewModel.getNewEid()
+  val event =
+      Event(
+          eid,
+          "unknown",
+          "unknown",
```

Comment:
Suggestion: change initial description value to an empty string for the sake of clarity when fetching data